tests/Unit/Instrumentation/AwsSdk/AwsSdkInstrumentationTest.php

```diff
@@ -37,6 +55,55 @@ public function testInstrumentationInit()
         );
     }
 
+    public function testSetXrayPropagator()
```

Comment:
this test doesn't do anything useful. both set and get are covered by the next test, though.